RELEASE_NOTES.md

```diff
@@ -19,7 +19,7 @@ information, see [SAT in CSM](operations/system_admin_toolkit/about_sat/SAT_in_C
 
 ### New hardware support
 
-* **IMPORTANT** Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
+* Systems with NVIDIA CPUs and GPUs.
```

Comment:
Since 1.6.1 is not yet released, should this either wait to merge or should we mention that NVIDIA CPU/GPU is unsupported in 1.6.0?